src/lang/std/engineConnection.ts

```diff
@@ -1366,14 +1361,11 @@ export class EngineCommandManager {
     callback,
   }: Subscription<T>): () => void {
     const localUnsubscribeId = uuidv4()
-    const otherEventCallbacks = this.subscriptions[event]
-    if (otherEventCallbacks) {
-      otherEventCallbacks[localUnsubscribeId] = callback
-    } else {
-      this.subscriptions[event] = {
-        [localUnsubscribeId]: callback,
-      }
+    if (!this.subscriptions[event]) {
+      this.subscriptions[event] = {}
     }
+    this.subscriptions[event][localUnsubscribeId] = callback
+
```

Comment:
This and same with `subscribeToUnreliable` below is no change to functionality, but Paul had changed it and it is neater.